lib/defaults/defaults.go

```diff
@@ -1155,6 +1155,11 @@ var (
 	// successfully.
 	// The exit code is used to prevent the agent service from restarting after shut down
 	CompletedOperationExitCode = 253
```

Comment:
My 2 cents: I don't tend to make 0 a success code, I tend to use 0 as UNKNOWN, since if the variable is uninitialized to 0 it will stand out and not accidentally get interpreted as success when it's actually uninitialized.

Reply:
It is a very specific success code - one that should have a specific effect on the service. One can manually stop the service which would also exit with 0 but will not be finished (as is implied by this code).